__tests__/app.test.js

```diff
@@ -97,6 +97,7 @@ describe("GET /api/articles/:article_id", () => {
               body: "I find this existence challenging",
               created_at: expect.any(String),
               votes: 100,
+              comment_count: 11,
```

Comment:
when you add a new feature you should create a new and separate test, not alter previous tests. If the original test failed with the changes it means it was a brittle test.